db/node/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "hopr-db-node"
+name = "hopr-utils-db-node"
```

Comment:
Same here, I'm not a fan of putting the `utils` prefix there. 
We can then reexport the crates from a `hopr-utils` übercrate.